### PR TITLE
Nemo/Added userFullName prop

### PR DESCRIPTION
**Pull request by nemodjuric**
Connected to #2421 

prefill

### CODE REVIEW

File: client/app/certification/reducers/index.js [hunk 1]
```diff
@@ -142,6 +142,10 @@ export const poaCorrectLocationToStr = function(poaCorrectInVacols, poaCorrectIn
   return null;
 };
 
+const certifyingOfficialName = function(certificationName, userName) {
```
Comment (amprokop) [inline client/app/certification/reducers/index.js]: This is a confusing bit of logic to have on the frontend. We should just decide on the backend what the correct value is for `certifyingOfficialName`, and ensure that is passed down.

File: app/views/certifications/v2.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <%= react_component("Certification", props: {
-  vacols_id: @certification.vacols_id
+  vacolsId: @certification.vacols_id, userFullName: current_user.full_name
```
Comment (amprokop) [inline app/views/certifications/v2.html.erb]: it is strange to pass this down as a separate property, considering it's a column on certification

File: client/app/certification/Certification.jsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ const EntryPointRedirect = connect(
   mapStateToProps
 )(UnconnectedEntryPointRedirect);
 
-const configureStore = (certification, form9PdfPath) => {
+const configureStore = (certification, form9PdfPath, userFullName) => {
```
Comment (amprokop) [inline client/app/certification/Certification.jsx]: don't need this change if we pass the user's full name down as part of the cert

File: client/app/certification/Certification.jsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ const configureStore = (certification, form9PdfPath) => {
   // eslint-disable-next-line no-underscore-dangle
   const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-  const initialData = mapDataToInitialState(certification, form9PdfPath);
+  const initialData = mapDataToInitialState(certification, form9PdfPath, userFullName);
```
Comment (amprokop) [inline client/app/certification/Certification.jsx]: don't need this change if we pass the user's full name down as part of the cert

File: client/app/certification/Certification.jsx [hunk 5]
```diff
@@ -160,7 +160,7 @@ export default class Certification extends React.Component {
     }
 
     { this.state.certification &&
-      <Provider store={configureStore(this.state.certification, this.state.form9PdfPath)}>
+      <Provider store={configureStore(this.state.certification, this.state.form9PdfPath, this.props.userFullName)}>
```
Comment (amprokop) [inline client/app/certification/Certification.jsx]: don't need this change if we pass the user's full name down as part of the cert

File: client/app/certification/reducers/index.js [hunk 6]
```diff
@@ -192,7 +196,7 @@ export const mapDataToInitialState = (certification, form9PdfPath) => ({
   certifyingOffice: certification.certifying_office,
   certifyingUsername: certification.certifying_username,
   certificationDate: certification.certification_date,
-  certifyingOfficialName: certification.certifying_official_name,
+  certifyingOfficialName: certifyingOfficialName(certification.certifying_official_name, userFullName),
```
Comment (amprokop) [inline client/app/certification/reducers/index.js]: don't need this change if we pass the user's full name down as part of the cert

File: client/app/certification/reducers/index.js [hunk 7]
```diff
@@ -164,7 +168,7 @@ const parseDocumentFromApi = (doc = {}, index) => ({
   isExactlyMatching: doc.serialized_vacols_date === doc.serialized_receipt_date
 });
 
-export const mapDataToInitialState = (certification, form9PdfPath) => ({
+export const mapDataToInitialState = (certification, form9PdfPath, userFullName) => ({
```
Comment (amprokop) [inline client/app/certification/reducers/index.js]: don't need this change if we pass the user's full name down as part of the cert

File: client/app/certification/reducers/index.js (hunk 1 above)
Comment (nemodjuric) [inline client/app/certification/reducers/index.js]: Certification field 'certifyingOfficialName' gets set after you click Continue on SignAndCertify Page. Before that is null.

Comment (amprokop) [inline client/app/certification/reducers/index.js]: yes, see my other comments — we should set it

**Pull request merged**